tests/test_process_node.py

```diff
@@ -412,14 +308,16 @@ def test_process_query_view_parity_contains(
 
         # Don't fuck with newlines due to a dgraph bug
         # https://github.com/dgraph-io/dgraph/issues/4694
-        if len(arguments) > 3 and "\n" not in arguments:
-            query.with_arguments(contains=arguments[: len(arguments) - 1])
-        if len(asset_id) > 3 and "\n" not in asset_id:
-            query.with_asset_id(contains=asset_id[: len(asset_id) - 1])
-        if len(image_name) > 3 and "\n" not in image_name:
-            query.with_image_name(contains=image_name[: len(image_name) - 1])
-        if len(process_name) > 3 and "\n" not in process_name:
-            query.with_process_name(contains=process_name[: len(process_name) - 1])
+        for prop in [arguments, image_name, process_name]:
+            hypothesis.assume(len(prop) > 3)
+            hypothesis.assume("\n" not in prop)
+            hypothesis.assume("\\" not in prop)
+
```

Comment:
since I'm @givening a text_dgraph_compat() - which is a bad name, I know - these assumes will never fail.

this is different behavior than previously, which would just skip doing that query entirely; maybe that's actually what we want

Reply:
No, this behavior is better.